src/languages.rs

```diff
@@ -2,12 +2,47 @@ extern crate uuid;
 
 use rocket::Response;
 use rocket::http::ContentType;
+use rocket::http::Status;
+
+use postgres::error::UNIQUE_VIOLATION;
 
 use std::io::Cursor;
 
 use db;
 use sentences::Sentence;
 
+#[post("/languages", format="text/plain", data="<iso639_3>")]
+fn create_language<'r>(
+    connection: db::DbConnection,
+    iso639_3: String,
+) -> Response<'r> {
+
+    let result = connection.query(
+        r#"
+        INSERT INTO language(iso639_3)
+        VALUES ($1)
+        "#,
+        &[&iso639_3],
+    );
+
+    match result {
+        Ok(_) => {},
```

Comment:
if we have only a case for for Err , shouldn't we have a if instead of a match ?

Reply:
I tried both. Eventually I decided to do the same as before, but yes we can replace it by `if`.